src/main/java/uk/gov/justice/digital/hmpps/csr/api/dto/ShiftNotificationDto.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.csr.api.dto
+
+import java.util.*
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import uk.gov.justice.digital.hmpps.csr.api.domain.ActionType
+import uk.gov.justice.digital.hmpps.csr.api.domain.ShiftType
+import uk.gov.justice.digital.hmpps.csr.api.model.ShiftNotification
+import java.time.Clock
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@ApiModel(description = "ShiftNotification")
+data class ShiftNotificationDto @JsonCreator constructor(
+        @ApiModelProperty(required = true, value = "User unique identifier", position = 1, example = "AB102CD")
+        @JsonProperty("quantumId")
+        val quantumId: String,
+
+        @ApiModelProperty(required = true, value = "When the shift notification was created", position = 2, example = "2020-04-20T17:45:55")
+        @JsonProperty("createdAt")
+        val createdAt: LocalDateTime,
```

Comment:
This actually isn't, its the date the stored procedure was run, I think we can drop it because its not helpful.